tests/bootstrap.php

```diff
@@ -0,0 +1,16 @@
+<?php
```

Comment:
Just curious: what is the motivation for this file? Is it trying to ensure that people use composer vs. some other dependency management tool?